packages/babel-plugin-proposal-decorators/test/fixtures/2022-03-ordering--to-es2015/accessor-static-accessor-initializers/exec.js

```diff
@@ -42,6 +42,9 @@ class A {
   accessor #d;
 }
 
+var nums = Array.from({ length: 30 }, (_, i) => i);
+expect(log).toEqual(nums);
```

Comment:
The test here becomes more stringent: we assert that certain log must be printed before the class is instantiated.